package exceptiontest;

import java.util.Random;

public class RuntimeExceptionTest {
	public static void main(String[] args) {
//		Random random = new Random();
//		int num = random.nextInt(3); // 0 1 2
//		int y = 10;
//		int z = y/num;
//		System.out.println(y + "/" + num + "=" + z);
		
//		Random random = new Random();
//		int num = random.nextInt(2)+1; // 0 1 2
//		int y = 10;
//		int z = y/num;
//		System.out.println(y + "/" + num + "=" + z);
		
		
	}
}
